generators/client/templates/vue/src/main/webapp/app/components/jhi-navbar/jhi-navbar.component.ts.ejs

```diff
@@ -54,6 +54,10 @@ export default class JhiNavbar extends mixins(Principal) {
   public changeLanguage(newLanguage) : void {
     this.refreshTranslation(newLanguage);
   }
+
+  public isActiveLanguage(key) {
```

Comment:
Please add types: `public isActiveLanguage(key: string): void {`

Reply:
Good point, not used to typescript :)